app/grants/models.py

```diff
@@ -900,3 +900,29 @@ class MatchPledge(SuperModel):
     def __str__(self):
         """Return the string representation of this object."""
         return f"{self.profile} <> {self.amount} DAI"
+
+class PhantomFunding(SuperModel):
+    """Define the structure of a PhantomFunding object.
+
+    For Grants, we have a fund we’re contributing on their behalf.  just having a quick button they can push saves all the hassle of (1) asking them their wallet, (2) sending them the DAI (3) contributing it.
+
+    """
+
+    round_number = models.PositiveIntegerField(blank=True, null=True)
```

Comment:
@frankchen07 this is the new model that will contain info about when people phantom fund.  

two ways we can manage this: 
1. you just build phantom funding into your existing reports
2. myself or engineering builds a `finalize_phantom_funding` job that runs once per round, at end of round, that 
a. closes out phantom funding for that round 
b. makes the phantom contribs into *actual* contributions (both on chain and in the DB) 
c. anything else??

cc @danlipert 4 other ideas

Reply:
1. I'm assuming the data model reflects this in a way where it's easy to build it into the reports. I think as long as it tell me the contributor, which grant(s) they've committed to and how much, it can be easily integrated into the report at the end of the CLR round.

2. from a product perspective, point number 2 is the trickier one, especially with https://github.com/gitcoinco/web/issues/3663 - since this is part of the product update that vitalik requested (a nifty little feature that tells us in "real" time, how much potential a contribution has in the LR context, and this needs all the contributions, real and phantom in order to be the most accurate). I might be overthinking it in the sense that if the data exists it can be used in the feature, and the `phantom_funding` job is a completely separate "payout" related action at the end.

In a related conversation, Dan suggested that we do some kind of simplified calculation and  cache the results so we can give them a range, since doing a live calculation with real data every time someone changes a number in an input box can be performance hell.

Reply:
1. yea it does that (tho 'how much' is inferred by 5/(num_votes_cast_by_contrib)
2. hmm.. yeah #3663 does make it more complicated.  ill add a note there about phantom funding.

agree with @danlipert s thoughts.